templates/includes/contact-us.html

```diff
@@ -10,141 +10,225 @@ <h2 class="p-modal__title" id="modal-title">Get your Kubeflow stack running</h2>
-            <input type="checkbox" id="workloads-private-cloud-Embedded">
-            <label for="workloads-private-cloud-Embedded">Embedded</label>
-            <input type="checkbox" id="workloads-private-cloud-Edge">
-            <label for="workloads-private-cloud-Edge">Edge</label>
+            <label class="p-checkbox" for="workloads-private-cloud-VMware">
+              <input class="p-checkbox__input"  type="checkbox" id="workloads-private-cloud-VMware">
+              <span class="p-checkbox__label">VMware</span>
+            </label>
+            <label class="p-checkbox" for="workloads-private-cloud-Bare-metal">
+              <input class="p-checkbox__input" type="checkbox" id="workloads-private-cloud-Bare-metal">
+              <span class="p-checkbox__label">Bare metal</span>
+            </label>
+            <label class="p-checkbox" for="workloads-private-cloud-OpenStack">
+              <input class="p-checkbox__input" type="checkbox" id="workloads-private-cloud-OpenStack">
+              <span class="p-checkbox__label">OpenStack</span>
+            </label>
+            <label class="p-checkbox" for="workloads-private-cloud-Workstations">
+              <input class="p-checkbox__input" type="checkbox" id="workloads-private-cloud-Workstations">
+              <span class="p-checkbox__label">Workstations</span>
+            </label>
+            <label class="p-checkbox" for="workloads-private-cloud-Embedded">
+              <input class="p-checkbox__input" type="checkbox" id="workloads-private-cloud-Embedded">
+              <span class="p-checkbox__label">Embedded</span>
+            </label>
+            <label class="p-checkbox" for="workloads-private-cloud-Edge">
+              <input class="p-checkbox__input" type="checkbox" id="workloads-private-cloud-Edge">
+              <span class="p-checkbox__label">Edge</span>
+            </label>
             <div class="js-other-container">
-              <input type="checkbox" class="js-other-container__checkbox" id="workloads-private-cloud-other">
-              <label for="workloads-private-cloud-other">Other</label>
-              <input type="text" id="workloads-private-cloud-other-specified" class="js-other-container__input" placeholder="Please specify" style="opacity: 0; margin-top: .25rem;" aria-label="Other private cloud platform">
+              <label class="p-checkbox" for="workloads-private-cloud-other">
+                <input type="checkbox" class="js-other-container__checkbox p-checkbox__input" id="workloads-private-cloud-other">
+                <span class="p-checkbox__label">Other</span>
+              </label>
+              <label for="workloads-private-cloud-other-specified">
+                <input type="text" id="workloads-private-cloud-other-specified" class="js-other-container__input" placeholder="Please specify" style="opacity: 0; margin-top: .25rem;" aria-label="Other private cloud platform">
+              </label>
             </div>
```

Comment:
There are two `others` checkboxes for this question so maybe remove this one?
<img width="588" alt="Screenshot 2022-02-02 at 11 43 58 am" src="https://user-images.githubusercontent.com/57550290/152147530-502e3672-dca1-4845-ae9b-dd3816a5c4df.png">

Reply:
These are different categories of produts, the first column talks about "bare metal" services, the second is a list of cloud services and the third as far as I know are chips, so they are different.

Reply:
ah that makes sense! sorry :)